velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -213,12 +213,21 @@ void WindowFuzzer::go() {
   auto startTime = std::chrono::system_clock::now();
   size_t iteration = 0;
 
+  auto vectorOptions = vectorFuzzer_.getOptions();
   while (!isDone(iteration, startTime)) {
     LOG(INFO) << "==============================> Started iteration "
               << iteration << " (seed: " << currentSeed_ << ")";
 
     auto signatureWithStats = pickSignature();
     signatureWithStats.second.numRuns++;
+    if (functionDataBoundary_.count(signatureWithStats.first.name) > 0) {
+      vectorOptions.dataBoundary =
+          functionDataBoundary_.at(signatureWithStats.first.name);
+
+    } else {
+      vectorOptions.dataBoundary = {true, true};
```

Comment:
nit: 
constexpr DataSpec kDefaultSpec{true, true}; // Define in VectorFuzzer.h
vectorOptions.dataSpec = kDefaultSpec; // Use here and in AggregationFuzzer.